server/createdb.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+
+from app import create_app
+app = create_app()
+app.app_context().push()
+from app import db
+db.create_all()
+print("just ran createdb.py")
+exit()
```

Comment:
This is a script right? Just maybe add it to a scripts folder, and use if __name__ == "__main__"